core/types/calls.cc

```diff
@@ -607,7 +607,7 @@ DispatchResult dispatchCallSymbol(const GlobalState &gs, const DispatchArgs &arg
                                 addedAutocorrect = true;
                             }
                         } else {
-                            const auto replacement = possibleSymbol->name.toString(gs);
+                            const auto replacement = possibleSymbol.data(gs)->name.toString(gs);
```

Comment:
Why does the `.data(gs)` get added here but removed in most other places?

Reply:
`possibleSymbol` is a `SymbolRef` now and not a `SymbolData`.